### PR TITLE
Fix PP issue

### PR DESCRIPTION
**Pull request by michaelbenayoun**
# What does this PR do?

As per title.

### CODE REVIEW

File: tests/test_trainers.py [hunk 1]
```diff
@@ -462,3 +463,9 @@ def test_without_packing(self, parallel_sizes, tmpdir):
 
     def test_with_packing(self, parallel_sizes, tmpdir):
         return self._test_sft_trainer(parallel_sizes, tmpdir, True)
+
+
+@is_trainium_test
```
Comment (dacorvo) [inline tests/test_trainers.py:468]: I think this is a WIP, right ?

**Review by dacorvo — APPROVED** [review]
LGTM, although this does not seem to fix the main issue with the neverending workflows.

File: tests/test_trainers.py (hunk 1 above)
Comment (michaelbenayoun) [reply tests/test_trainers.py:468]: No it's just that pytest fails with error code 5 when it does not collect staging and trainium tests. So I just created a dummy test.

**Comment by michaelbenayoun** [comment]
Yes very hard to debug the neverending issue.

**Pull request merged**